backend/apps/sandbox/serializers.py

```
@@ -158,4 +158,22 @@ class Meta:
+from .models import MaintainerScenario, MaintainerEvaluation
+
+class MaintainerScenarioSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MaintainerScenario
+        fields = ['id', 'title', 'description', 'original_code', 'flawed_code', 'diff_content', 'required_findings', 'created_at']
+        read_only_fields = ['id', 'created_at']
+
+class MaintainerEvaluationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MaintainerEvaluation
+        fields = ['id', 'scenario', 'user', 'submitted_comments', 'score', 'passed', 'created_at']
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)